classes/framework.php

```diff
@@ -176,10 +176,77 @@ public static function printMessages($type, $messages) {
+                'File "%filename" not allowed. Only files with the following extensions are allowed: %files-allowed.' => get_string('invalidfile', 'hvp'),
+                'Invalid selected option in multi-select.' => get_string('invalidmultiselectoption', 'hvp'),
+                'Invalid selected option in select.' => get_string('invalidselectoption', 'hvp'),
+                'H5P internal error: unknown content type "@type" in semantics. Removing content!' => get_string('invalidsemanticstype', 'hvp'),
+                'Library used in content is not a valid library according to semantics' => get_string('invalidsemantics', 'hvp'),
+                'Copyright information' => get_string('copyrightinfo', 'hvp'),
+                'Title' => get_string('titole', 'hvp'),
+                'Author' => get_string('autor', 'hvp'),
+                'Year(s)' => get_string('years', 'hvp'),
+                'Source' => get_string('source', 'hvp'),
+                'License' => get_string('license', 'hvp'),
+                'Undisclosed' => get_string('undisclosed', 'hvp'),
+                'Attribution 4.0' => get_string('attribution', 'hvp'),
+                'Attribution-ShareAlike 4.0' => get_string('attributionsa', 'hvp'),
+                'Attribution-NoDerivs 4.0' => get_string('attributionnd', 'hvp'),
+                'Attribution-NonCommercial 4.0' => get_string('attributionnc', 'hvp'),
+                'Attribution-NonCommercial-ShareAlike 4.0' => get_string('attributionncsa', 'hvp'),
+                'Attribution-NonCommercial-NoDerivs 4.0' => get_string('attributionncnd', 'hvp'),
+                'General Public License v3' => get_string('gpl', 'hvp'),
+                'Public Domain' => get_string('pd', 'hvp'),
+                'Public Domain Dedication and Licence' => get_string('pddl', 'hvp'),
+                'Public Domain Mark' => get_string('pdm', 'hvp'),
+                'Copyright' => get_string('copyright', 'hvp'),
+                'Unable to create directory.' => get_string('unabletocreatedir', 'hvp'),
+                'Unable to get field type.' => get_string('unabletogetfieldtype', 'hvp'),
+                "File type isn't allowed." => get_string('filetypenotallowed', 'hvp'),
+                'Invalid field type.' => get_string('invalidfieldtype', 'hvp'),
+                'Invalid image file format. Use jpg, png or gif.' => get_string('invalidimageformat', 'hvp'),
+                'File is not an image.' => get_string('filenotimage', 'hvp'),
+                'Invalid audio file format. Use mp3 or wav.' => get_string('invalidaudioformat', 'hvp'),
+                'Invalid video file format. Use mp4 or webm.' => get_string('invalidvideoformat', 'hvp'),
+                'Could not save file.' => get_string('couldnotsave', 'hvp'),
+                'Could not copy file.' => get_string('couldnotcopy', 'hvp')
```

Comment:
Shouldn't the last ten strings be in `lang/en/hvp.php` as well? I get lots of notices.